packages/web-app-unzip/types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*?url' {
+  const content: string
+  export default content
+}
```

Comment:
Meeeeeeeh... isnt this included in vite types?

Reply:
It should... but it doesn't work, and I can't get behind it.